Bottom: 7fe5b6815e28461fa033e596c8d29b8717a40bb8
Top:    a7d059d0b7ef241febde0a05eb86ab0f0acf6aef
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-10 17:03:33 -0500

Refresh of add-buildbot-role

---
diff --git a/roles/buildbot-master/defaults/main.yml b/roles/buildbot-master/defaults/main.yml
index cd939d11..48e5ec18 100644
--- a/roles/buildbot-master/defaults/main.yml
+++ b/roles/buildbot-master/defaults/main.yml
@@ -25,7 +25,7 @@ buildbot_https_port: 8443
 # The hostname passed as an envirnment variable into the container
 buildbot_ip_addr: '127.0.0.1'
 buildbot_hostname: buildbot.example.com
-buildbot_port_args: 
+buildbot_port_args:
   - '{{buildbot_http_port}}'
   - '{{buildbot_https_port}}'
   - '{{buildbot_worker_port}}'
